UseCases.tex

```diff
@@ -1,48 +1,73 @@
 \subsection{Event-List Data and Responses}
 
 
-\subsubsection{Use Case --- Search for event lists surrounding Sgr A* for morphological studies}
+\subsubsection{Use Case --- Search for event lists surrounding Sgr A*, for exemple for an X-ray morphological study}
 
-{\em Identify all HEA event lists encompassing Sgr~A* for initial selection for subsequent morphological studies.  Since a large number (at least several hundred) of observations are expected to match this request and the focus is on morphological studies, only the event lists and not the event bundles are desired.\/}
+{\em Identify all \gls{HE} event lists encompassing Sgr~A* for initial selection for subsequent X-ray morphological studies.  Since the focus is on X-ray morphological studies, only the event lists and not the event bundles are desired.\/}
 
 \medskip
 \noindent Find all datasets satisfying:
 \begin{enumerate}[(i)]
-  \item Target name = ``Sgr A*'' or position inside 30 arcmin from (266.4168, $-29.0078$).
-  \item dataproduct\_type = ``event-list''.
+  \item Target name = ``Sgr A*'' or position inside 30 arcmin from (266.4168, $-29.0078$)
+  \item dataproduct\_type = ``event-list''
 \end{enumerate}
 
 \begin{verbatim}
 SELECT * FROM ivoa.obscore
-NATURAL JOIN  ivoa.obscore-hea 
+NATURAL JOIN  ivoa.obscore_he 
 WHERE
-(target_name = `Sgr A*' OR
+(target_name = 'Sgr A*' OR
 CONTAINS(POINT(s_ra, s_dec), CIRCLE, 266.4168, -29.0078, 0.5) = 1) 
-AND (dataproduct_type EQ `event-list')
+AND (dataproduct_type = 'event-list')
 \end{verbatim}
 
 
-\subsubsection{Use Case --- Search for event bundles that include Cas A for spectrophotometric evolution studies}
+\subsubsection{Use Case --- Search for event bundles that include Cas A for X-ray spectrophotometric evolution studies}
 
-{\em Identify all event bundles that include the Cas A SNR and have at least 1 million events for subsequent spectrophotometric studies of the SNR expansion.  Since only a few observations are expected to match this request and because the focus is on spectrophotometric studies, the event bundles that include the responses or the ancillary products used to make the responses are required.\/}
+{\em Identify all event bundles that include the Cas A SNR and have at least 1 million events for subsequent spectrophotometric studies of the SNR expansion. Since only a few observations are expected to match this request and because the focus is on X-ray spectrophotometric studies, the event bundles that include the responses or the ancillary products used to make the responses are required.\/}
 
 \medskip
 \noindent Find all datasets satisfying:
 \begin{enumerate}[(i)]
-  \item Target name = ``Cas A'' or position inside 10 arcmin from (350.8584, $+58.8113$).
-  \item dataproduct\_type = ``event-bundle''.
+  \item Target name = ``Cas A'' or position inside 10 arcmin from (350.8584, $+58.8113$)
+  \item dataproduct\_type = ``event-bundle''
 \end{enumerate}
 
 \begin{verbatim}
```

Comment:
Tip: if you want this to look nicer, use the minted package and then do:

```latex
% in the header
\usepackage{minted}
\usemintedstyle{friendly}

...

\begin{minted}{sql}
SELECT * FROM ivoa.obscore
NATURAL JOIN  ivoa.obscore_he 
WHERE
  (target_name = 'Cas A'  OR CONTAINS(POINT(s_ra, s_dec), CIRCLE, 350.8584, +58.8113, 0.16667) = 1) 
  AND (dataproduct_type = 'event-bundle')
  AND (ev_xel >= 1000000)
\end{minted}

```

Then you get nice syntax highliting like:

```sql
SELECT * FROM ivoa.obscore
NATURAL JOIN  ivoa.obscore_he 
WHERE
    (target_name = 'Cas A'  OR  CONTAINS(POINT(s_ra, s_dec), CIRCLE, 350.8584, +58.8113, 0.16667) = 1) 
    AND (dataproduct_type = 'event-bundle')
    AND (ev_xel >= 1000000)
```

Reply:
Great... I will make the editing in this direction ;)